esiclient/v1/node_network.py

```diff
@@ -175,6 +175,9 @@ def take_action(self, parsed_args):
         if parsed_args.network:
             network = neutron_client.find_network(parsed_args.network)
             port = None
+            if network is None:
+                raise exceptions.CommandError(
+                    "ERROR: This is not a network name")
```

Comment:
I'd say something like 'Unknown network' instead, since I think the user can also pass in a network UUID.

Reply:
I would say something like - "ERROR: This is not a network name or UUID" because then it aligns with the error message we are passing for the wrong port name or trunk name below.

Reply:
To be honest, I'd change it because whatever the user passes in *is* a network name; it's just not necessarily one that's in the system.